app/models.py

```diff
@@ -347,6 +350,18 @@ class Notification(db.Model):
     status = db.Column(
         db.Enum(*NOTIFICATION_STATUS_TYPES, name='notify_status_types'), nullable=False, default='sending')
     reference = db.Column(db.String, nullable=True, index=True)
+    _personalisation = db.Column(db.String, nullable=True)
```

Comment:
i'd rather we give the column the name `personalisation` - if memory serves me you should just be able to write `_personalisation = db.Column('personalisation', db.String, nullable=True)` and that will keep it mapped to `_personalisation` in the ORM but `personalisation` in the actual DB